src/main/rx/crsf.c

```diff
@@ -241,8 +242,15 @@ STATIC_UNIT_TESTED uint8_t crsfFrameStatus(rxRuntimeConfig_t *rxRuntimeConfig)
             rxLinkStatistics.uplinkTXPower = crsfPowerStates[linkStats->uplinkTXPower];
             rxLinkStatistics.activeAnt = linkStats->activeAntenna;
 
-            if (rxLinkStatistics.uplinkLQ > 0)
-                lqTrackerSet(rxRuntimeConfig->lqTracker, scaleRange(constrain(rxLinkStatistics.uplinkRSSI, -120, -30), -120, -30, 0, RSSI_MAX_VALUE));
+            if (rxLinkStatistics.uplinkLQ > 0) {
+                int16_t uplinkStrength;   // RSSI dBm converted to %
+                uplinkStrength = constrain((100 * ((osdConfig()->rssi_dbm_max - osdConfig()->rssi_dbm_min) * (osdConfig()->rssi_dbm_max  - osdConfig()->rssi_dbm_min)) - (osdConfig()->rssi_dbm_max  - rxLinkStatistics.uplinkRSSI) * (100 * (osdConfig()->rssi_dbm_max  - rxLinkStatistics.uplinkRSSI))) / ((osdConfig()->rssi_dbm_max  - osdConfig()->rssi_dbm_min) * (osdConfig()->rssi_dbm_max  - osdConfig()->rssi_dbm_min)),0,100);
```

Comment:
If you put this line in the if/else construct below it doesn't get calculated when RSSI is outside the 0-100% range.

Reply:
Also, this line can be simplified to 
```
uplinkStrength = constrain(100 * (1 - (sq(osdConfig()->rssi_dbm_max  - osdConfig()->rssi_dbm_min) / sq(osdConfig()->rssi_dbm_max - rxLinkStatistics.uplinkRSSI))), 0, 100);;
```

Reply:
Thanks for the input. I will update with sq() but there's no need to square everything

Reply:
I only simplified the calculation by canceling terms.